src/services/stag/SchedulesService.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * @typedef (object) StudentService
```

Comment:
Komentář ke třídě by neměl `typedef`-inovat nový typy. Obzvlášť potom ne neodpovídajících jmen.
Nevím jak se to zachová ale myslel bych že nijak pěkně, stačí smazat tenhle doc-block.